docs/source/examples/request/timelines/post/extensive_inputs.json

```diff
@@ -250,16 +278,29 @@
                       }
                     ]
                   }
-                ]
+                ],
+                "hvacDistributionSystems": {
+                  "airDistributionSystems": [
+                    {
+                      "id": "HVACDistributionFurnaceAndCentralAC1",
+                      "ducts": [
+                        {
+                          "id": "HVACDistributionFurnaceAndCentral1Supply",
+                          "leakageValue": 0.1
+                        },
+                        {
+                          "id": "HVACDistributionFurnaceAndCentral1Return",
+                          "leakageValue": 0.1
+                        }
+                      ]
+                    }
+                  ]
```

Comment:
@kellyatradiantlabs I sent this request example through our current schemas and it failed. Right now, our schemas require us to provide all the fields for `airDistributionSystems[*]` and `airDistributionSystems[*].ducts[*]`. Could you remind me if we need to set some or all fields as optional? Or is it mandatory to define them if we provide either of these objects? Does it depend on whether the distribution system is provided in the base or improved building?

Here's the code for both schemas:

- [HVACAirDistributionSystem](https://github.com/RadiantLabs/core-modeling-api/blob/revisit-fixtures/libs/modeling-common/src/radiantlabs/modeling/schema/building/hvac/distribution/air.clj)
- [Duct](https://github.com/RadiantLabs/core-modeling-api/blob/revisit-fixtures/libs/modeling-common/src/radiantlabs/modeling/schema/building/hvac/distribution/duct.clj)

In both cases, all properties are mandatory, either for base or improved buildings.

PS: Just as a side note, I attach here the server JSON response, informing about the errors.

[request_errors.zip](https://github.com/RadiantLabs/modeling-api-docs/files/11923943/request_errors.zip)

Reply:
@joshuamzm ahh good question. See this table for which values should/should not be required: https://github.com/RadiantLabs/modeling-api-docs/blob/main/docs/source/api.rst#hvac-air-distribution-systems. If a system is defined in `baseBuilding` with a specific identifier, I was envisioning that any references to that system in the improvedBuilding would be revising that existing system. In the example called out here, the existing air distribution system is being kept in the new building and just the leakage values for both the supply and return ducts is getting reduced (via a duct sealing measure).